apps/gateway-ui/src/components/WithdrawCard.tsx

```diff
@@ -96,35 +96,16 @@ export const WithdrawCard = React.memo(function WithdrawCard({
       });
   };
 
+  const description = `${t('withdraw-card.total_bitcoin')} ${
+    withdrawObject.amount / 100000
+  } ${t('common.btc')}`;
+
   return (
-    <Box w='100%'>
-      <GatewayCard>
-        <Stack spacing='4px' h='100%'>
-          <Text
-            fontSize='lg'
-            fontWeight='600'
-            color={theme.colors.gray[900]}
-            fontFamily={theme.fonts.body}
-          >
-            {t('withdraw-card.card-header')}
-          </Text>
-          <Text
-            fontSize='md'
-            color={theme.colors.gray[600]}
-            fontFamily={theme.fonts.body}
-          >
-            {t('withdraw-card.total_bitcoin')} {withdrawObject.amount / 100000}{' '}
-            {t('common.btc')}
-          </Text>
-          <Text
-            cursor='pointer'
-            fontSize='sm'
-            color={theme.colors.blue[600]}
-            fontFamily={theme.fonts.body}
-          >
-            {t('withdraw-card.withdraw_all')}
-          </Text>
-        </Stack>
+    <>
```

Comment:
I don't see this pattern in the other cards?

Reply:
This is due to the `<ConfirmWithdrawModal />` at the bottom. React components can only render a single node, which is why this was previously wrapped in a `<Box />`. However, that adds an unnecessary extra `<div>` to the markup. Using a `<>` React Fragment means we return one React node, but it doesn't render excess markup in the DOM that can effect layout.